error.handling.js

```diff
@@ -1,5 +1,5 @@
 exports.sqlErrors = (err, req, res, next) => {
-  if (err.code) {
+  if (err.code === "22P02") {
     return res.status(404).send({ msg: "Not found" });
```

Comment:
Did we encounter a 22P02 yet? And would we want a 404 if we did? Food for thought!